dev-libs/re2/re2-0.2018.02.01.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="A Regular Expression Library"
+DESCRIPTION="RE2 is a fast, safe, thread-friendly alternative to backtracking \
+regular expression engines like those used in PCRE, PERL and Python. It is a \
+C++ library."
+HOMEPAGE="https://github.com/google/re2"
+COPYRIGHT="2009 The RE2 Authors"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+RE_VERSION="${portVersion//./-}"
+RE_VERSION="${RE_VERSION#0-}"
```

Comment:
You could just name the recipe `re2-20180201.recipe` and avoid all of this together.